recipe/meta.yaml

```diff
@@ -1,27 +1,44 @@
-{% set version = "3.0.0b2.post2" %}
+{% set version = "2.6.1" %}
```

Comment:
I recommend to create a different branch for that version. Like we did for the `gdal-feedstock`.